specs/merge/beacon-chain.md

```diff
@@ -126,6 +127,7 @@ class ExecutionPayloadHeader(Container):
     timestamp: uint64
     receipt_root: Bytes32
     logs_bloom: ByteVector[BYTES_PER_LOGS_BLOOM]
+    randao: Bytes32
```

Comment:
Not sure if we should call it `randao` or `randao_mix`. There is a mix-hash in eth1, which is completely different, part of ethash, so that may be confusing. That mix-hash is not part of the seal (see https://github.com/ethereum/go-ethereum/blob/eff998effbf466d4acbe599583dd8ec75f2af6a1/consensus/ethash/consensus.go#L606), but is part of the header and old consensus checks (https://github.com/ethereum/go-ethereum/blob/eff998effbf466d4acbe599583dd8ec75f2af6a1/consensus/ethash/consensus.go#L566)

Reply:
`mix_hash` is a part of PoW seal but not a part of seal hash. I think that `randao` is a short and clean name, and it is less restrictive semantically. In the context of beacon chain `randao_mix` is associated with concrete value. And I would leave an ability to add something atop of `randao_mix` before it's passed to the payload in the future.